src/components/Home.jsx

```diff
@@ -116,8 +120,11 @@ const Home = ({ currentLanguage, dispatch, history, language, rtl, translate })
             <div className="home-page__buttons">
               <div>
                 <a
-                  className="button button__dark-disabled"
-                  // href={`${classifyPath}${c.hebrewKeyword}`}
+                  className={classnames('button', {
+                    'button__dark-disabled': !adminMode
+                  })}
+                  href={`${classifyPath}${c.hebrewKeyword}`}
+                  rel="noopener noreferrer"
```

Comment:
Minor note: since the <a> tag doesn't support the disabled attribute, and this <a> tag has a href, users can still access the link. They can't click the link (thanks to .button__dark-disabled's pointer-events:none) but they can tab-navigate and open the link via keyboard.